root/static/scripts/common/components/ArtistCreditLink.js

```diff
@@ -23,13 +22,13 @@ const ArtistCreditLink = ({artistCredit, showDeleted = true, ...props}) => {
         <EntityLink
           content={credit.name}
           entity={artist}
-          key={i}
+          key={artist.gid}
```

Comment:
Using the artist MBID as a key is incorrect, because an artist credit can link to the same artist multiple times (i.e. it doesn't uniquely identify each artist credit name). Using the index (position) as the key was correct in this case.